tooling/csv-to-questions.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
```

Comment:
Your Linux distribution is [broken](https://www.python.org/dev/peps/pep-0394/), you know? 😉

Anyway, the script in master right now should work with both python 2 and python 3.

Reply:
Your own link states the following:

> however, end users should be aware that python refers to python3 on at least Arch Linux (that change is what prompted the creation of this PEP), so python should be used in the shebang line only for scripts that are source compatible with both Python 2 and 3.

Coincidentally, my distribution is Arch Linux. So working as intended :smile: 
but not problem anyway since you updated the script